src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/LocationType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship
+{
+    public enum LocationType
```

Comment:
Would this enum itself make more sense in the Domain layer as it is business behaviour?

Reply:
There is a conversion between domain and application which will show up when more types are introduced. Employer and Specific both will be mapped to standard in domain.

Reply:
👍